pkg/cmd/prebuild/update.go

```diff
@@ -139,12 +183,15 @@ var prebuildUpdateCmd = &cobra.Command{
 
 			views.RenderViewBuildLogsMessage(buildId)
 		}
+
 		return nil
 	},
 }
 
-var runOnUpdateFlag bool
-
 func init() {
+	prebuildUpdateCmd.Flags().StringVar(&branchFlag, "branch", "", "Git branch for the prebuild")
```

Comment:
Add shorthands (e.g. `StringVarP`) for all flags except `run`

`branch` => `b`
`retention` => `r`
`commit-interval` => `c`
`trigger-files` => `t`

Reply:
updated it